pkg/scheduler/framework/interface_test.go

```diff
@@ -66,12 +66,12 @@ func TestStatus(t *testing.T) {
 		},
 		{
 			name:              "nil status",
-			status:            NewStatus(None, "nil error"),
-			expectedCode:      None,
-			expectedMessage:   "nil error",
-			expectedIsSuccess: false,
+			status:            NewStatus(Success, ""),
```

Comment:
since the design of the scheduler seems to make Success even if nil, we decided to create a Success Status.

Reply:
please revert this. It was intentional

Reply:
ok. 
thx for reviewed